devel/python/tests/ert/enkf/test_local_config.py

```diff
@@ -0,0 +1,56 @@
+import os
+import os.path
+from ert.test import ExtendedTestCase
+from ert.test import TestAreaContext
+from ert.enkf.enkf_main import EnKFMain
+
+class LocalConfigTest(ExtendedTestCase):
+    
+    def setUp(self):
+        self.case_directory = self.createTestPath("local/simple_config/")
+
+
+    def test_local_config( self ):
+        with TestAreaContext("enkf_test", store_area=True) as work_area:
```

Comment:
Here you can alternatively use `ErtTestContext` - then you do not need to manually copy the directory, and the main instance is automagically instantiated:
```
with ErtTestContext("manual_load_test", self.config_file, store_area = True) as test_context:
   main = test_context.getErt()
   ....
```
If using `ErtTestContext` the whole rest of the test should be in the `with ... :` scope.

I guess the `store_area = True` is leftover from debugging? Should be set to `False`


```